compiler/src/dotty/tools/dotc/core/NameOps.scala

```diff
@@ -27,7 +27,7 @@ object NameOps {
      */
     def apply(s: String)(implicit ctx: Context): String = {
       val marker = "$$$$"
-      val limit: Int = ctx.settings.XmaxClassfileName.value
+      final val LIMIT: Int = 240
       val MaxNameLength = (limit - 6) min 2 * (limit - 6 - 2 * marker.length - 32)
```

Comment:
you renamed limit to LIMIT but didn't update the usages here with the new name.

Reply:
The first commit, i see you had a second commit where you reverted that but also removed final, which is unrelated. In any case, your two commits should be squashed into one commit to keep the git history clean.

Reply:
Isn't squash merge allowed for this repo? I would think minor pulls like this would get squashed durin merge anyway

Reply:
@smarter squashed the commits fyi, cleaner now